documentation/user_docs/docs/manual/Manipulating_and_extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -33,18 +33,17 @@ There are optional fields too:
 
 **Non-orthogonal axes case**
 
-In the case when the crystal lattice is not orthogonal there are two options for viewing the data. In the default case the ``proj`` is specified in the same way as above, however remember that, for example, if proj.u = (1,0,0) and proj.v=(0,1,0), but (1,0,0) and (0,1,0) are not orthongonal in reciprocal space, the second viewing axes will actually be the vector orthogonal to proj.u in the plane of proj.u and proj.v. The third axis is then the cross product as before.
+In the case when the crystal lattice is not orthogonal there are two options for viewing the data. In the default case the ``proj`` is specified in the same way as above, however remember that, for example, if proj.u = (1,0,0) and proj.v=(0,1,0), but (1,0,0) and (0,1,0) are not orthogonal in reciprocal space, the second viewing axes will actually be the vector orthogonal to proj.u in the plane of proj.u and proj.v. The third axis is then the cross product as before.
 
 You may optionally choose to use non-orthogonal axes, as in the following example. Specify proj.u = (1,0,0), proj.v=(0,1,0), ``proj.nonorthogonal = true`` (and optionally specify the third projection axis by setting ``proj.w=[0,0,1]``). This forces the axes to be the ones you define, even if they are not orthogonal. Beware the plots that are produced plot them as orthogonal axes so any features may be skewed. However, it does make reading the location of a feature in a two-dimensional Q-Q plot straightforward, which is the main reason for doing this.
 
 **Binning arguments**
 
-- ``p1_bin``, ``p2_bin``, ``p2_bin`` specify the binning / integration arguments for the Q-axes. Each can independently have one of three different forms:
-   - If a single number (scalar) is given then that axis will be a plot axis and the bin width will be the number you specify. The lower and upper limits will be determined by the extent of the data along that direction.
-   - If you specify a vector with three components then that axis will be a plot axis with the first and last components specifying the lower and upper limits of data to be cut, and the middle component specifies the bin width.
+- ``p1_bin``, ``p2_bin``, ``p2_bin`` ``p4_bin``  specify the binning / integration arguments for the Q&Energy axes. Each can independently have one of three different forms:
```

Comment:
think the 3rd item should be p3_bin and should have a comma after it.

Q & Energy should have spaces around &

Are all of these dependent properties of the projection? Should probably have "proj." in front of them.